feat(final review list survey and admin): add an extra edit meal button and the one checkbox option

mucc001 opened the pull request.

lukashroch commented on apps/survey/src/components/layouts/meal-list/desktop/reviewed-list.vue:
```diff
@@ -12,14 +25,36 @@
-          v-if="review === 'checkbox'"
-          v-model="reviewed"
-          class="review-checkbox__checkbox pl-3"
-          label="Reviewed"
```
translate -> add string to prompts.json

lukashroch commented on apps/survey/src/components/layouts/meal-list/desktop/reviewed-list.vue:
```diff
@@ -12,14 +25,36 @@
+            <v-checkbox
+              v-model="reviewed"
+              class="review-checkbox__checkbox"
+              label="Reviewed, nothing to add/change"
```
translate -> add string to prompts.json

lukashroch commented on apps/survey/src/components/layouts/meal-list/desktop/reviewed-list.vue:
```diff
@@ -12,14 +25,36 @@
+        v-if="review === 'onecheckbox'"
+        v-model="reviewed"
+        class="pl-3 review-checkbox__checkbox"
+        label="Reviewed, nothing to add/change"
```
translate -> add string to prompts.json

lukashroch commented on apps/survey/src/components/layouts/meal-list/mobile/reviewed-list.vue:
```diff
@@ -13,14 +13,36 @@
-              v-if="review === 'checkbox'"
-              v-model="reviewed"
-              class="review-checkbox__checkbox pl-3"
-              label="Reviewed"
```
translate -> add string to prompts.json

lukashroch commented on apps/survey/src/components/layouts/meal-list/mobile/reviewed-list.vue:
```diff
@@ -13,14 +13,36 @@
+                <v-checkbox
+                  v-model="reviewed"
+                  class="review-checkbox__checkbox"
+                  label="Reviewed, nothing to add/change"
```
translate -> add string to prompts.json

lukashroch commented on apps/survey/src/components/layouts/meal-list/mobile/reviewed-list.vue:
```diff
@@ -13,14 +13,36 @@
+            v-if="review === 'onecheckbox'"
+            v-model="reviewed"
+            class="pl-3 review-checkbox__checkbox"
+            label="Reviewed, nothing to add/change"
```
translate -> add string to prompts.json

lukashroch reviewed (changes requested):
@mucc001 can you please translate the strings?

They should go to prompt.js, to the `submit-prompt`, so the prompt is translatable without hardcoded text.

Also I don't see for admin to be able to select `onecheckbox`, you'll just need to add it to https://github.com/MRC-Epid-it24/intake24/blob/master/packages/common/src/prompts/prompts.ts#L105C1-L105C1

mucc001 commented:
> @mucc001 can you please translate the strings?
> 
> They should go to prompt.js, to the `submit-prompt`, so the prompt is translatable without hardcoded text.
> 
> Also I don't see for admin to be able to select `onecheckbox`, you'll just need to add it to https://github.com/MRC-Epid-it24/intake24/blob/master/packages/common/src/prompts/prompts.ts#L105C1-L105C1

Hi Lukas. Thank you for the comments. I have moved the label to recall.json. In addition, I added the relevant code to the admin.
